blog/snippets/playwright/basic-screenshot-clipped.js

```diff
@@ -0,0 +1,21 @@
+const options = {
+  path: "clipped_screenshot.png",
+  fullPage: false,
+  clip: {
+    x: 5,
+    y: 60,
+    width: 240,
+    height: 40,
+  },
+};
+
+const { chromium } = require("playwright");
+
+(async () => {
+  const browser = await chromium.launch();
+  const page = await browser.newPage();
```

Comment:
I'm not fully familiar with playwright syntax, but isn't a `context` required in order to create a page?